routing.rst

```diff
@@ -794,6 +794,43 @@ parameter:
     To give a ``null`` default value to any parameter, add nothing after the
     ``?`` character (e.g. ``/blog/{page?}``).
 
+Priority Parameter
+~~~~~~~~~~~~~~~~~~~~
+
+.. versionadded:: 5.1
+
+    The ``priority`` parameter was introduced in Symfony 5.1
+
+Imagine that your application has a catch-all route in some cases this may be
+at the beginning of your routing collection and may have some undesirable effects.
+A ``priority`` optional parameter is available in order to let you choose the order of your routes,
+and this one is only available when using annotations to define routes.
+
+.. code-block:: php-annotations
+
+    // src/Controller/BlogController.php
+        namespace App\Controller;
+
+        use Symfony\Bundle\FrameworkBundle\Controller\AbstractController;
+        use Symfony\Component\Routing\Annotation\Route;
+
+        class BlogController extends AbstractController
+        {
+            // ...
+
+            /**
+             * @Route("/blog/{slug}", name="blog_show", priority=1)
```

Comment:
```suggestion
             * @Route("/{greedyParam}", name="greedy_route", priority=-2048)
```
Would allow to define it at last after any other routes.

Reply:
I would also keep this blog example below with a comment saying that it can be matched thanks to a default higher priority